pkg/ssh/utils.go

```diff
@@ -21,6 +21,7 @@ func Validate(user *url.Userinfo, path string, port int, identity string) (*conf
 	if strings.Contains(path, "/run") {
```

Comment:
Non-blocking, about all of `Validate`:

This kind of parsing/transformation function, overall, seems to me like a pretty good candidate for writing fairly exhaustive unit tests: it is
- non—trivial, with quite a few code paths and, apparently, corner cases (assuming they can’t be _eliminated_ in the first place)
- very isolated, so cheap to test